bolt-service/scripts/setup.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+BOLT_SERVER_PACKAGE=pe-bolt-server_0.22.0.7.g3257f59-1bionic_amd64.deb
+
+# Install Puppet-Agent and Bolt Server
+apt update -y
+apt install -y curl vim
+mkdir /tmp/bolt && pushd /tmp/bolt
+# Setup Puppet Agent Repo
+curl -O http://nightlies.puppet.com/apt/puppet6-nightly-release-bionic.deb && apt install -y ./puppet6-nightly-release-bionic.deb && rm ./puppet6-nightly-release-bionic.deb && apt update -y
+
+# Install PE Package
+curl -O https://artifactory.delivery.puppetlabs.net/artifactory/debian__local/pool/development/pe-bolt-server/3257f5940e3d4cebfa60ff9a06540f3b689020a6/ubuntu-18.04/${BOLT_SERVER_PACKAGE}
```

Comment:
This also seems very hard coded - is there a way to build this URL so that we don't have to update every release?

Reply:
Ideally we wouldn't use the package at all, systemctl doesn't actually run on most containers, so we get errors in the output due to service scripts.   It would be great to install the pe-bolt-server from source.   I'm happy to make changes if someone can give me guidance - so ideally if we build a container here it would reflect current codeline.   Is there a way to gem install from source?